tests/user_commands/management/commands/required_option.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand, CommandError
+
+
+class Command(BaseCommand):
+    help = "Be very demanding with parameters"
```

Comment:
I don't think this is needed. Or `requires_system_checks` or the non-required arguments -- try to simplify this as much as possible and remove unused things like `if options['verbosity'] > 0:` (I assume this was copied from another command).